src/xenia/base/clock_win.cc

```diff
@@ -14,6 +14,36 @@
 namespace xe {
 
 uint64_t Clock::host_tick_frequency() {
+#if XE_ARCH_AMD64 && XE_COMPILER_MSVC
```

Comment:
Does this function still run multiple times or just once per session?

1. If it runs multiple times, this probably only needs to be checked once, perhaps via a lambda populating a static variable (so it's thread safe).
2. If it runs only once, path using `QueryPerformanceFrequency` doesn't need to have its frequency as a static variable. Additionally, that path is currently not thread safe, as concurrent access may result in concurrent writes/reads to this non-atomic variable, which is UB.

Reply:
It runs at startup and every time time scaling is changed (+/-).
Just removed the `static` modifier.